src/helpers/fiware-token.helper.ts

```diff
@@ -15,8 +17,16 @@ export class AuthenticationTokenProvider {
     constructor(private httpService: HttpService, @Inject(CACHE_MANAGER) private cacheManager: Cache) {
     }
 
+    async clearConfig(config: FiwareDataTargetConfiguration): Promise<void> {
+        if (config.clientId) {
+            this.logger.debug(`AuthenticationTokenProvider clearing token for ${config.clientId}`)
```

Comment:
Any guidance when logging is considered to add value?

Reply:
On second thought, the log here is in line with the code style. My bad.
It's of course very subjective and project-dependent. Personally, I limit debug-level logs to avoid cluttering the logs.